Optimize the annotation of worker-size to avoid ambiguity (#783)
pingcap/docs-dm: en/task-configuration-file-full.md

# ----------- Global setting -----------
## ********* Basic configuration *********
name: test                      # The name of the task. Should be globally unique.
task-mode: all                  # The task mode. Can be set to `full`(only migrates full data)/`incremental`(replicates binlogs synchronously)/`all` (replicates both full data and incremental binlogs).
shard-mode: "pessimistic"       # The shard merge mode. Optional modes are ""/"pessimistic"/"optimistic". The "" mode is used by default which means no specific mode is required. If the task is a shard merge task, set it to the "pessimistic" mode. 
                                # After understanding the principles and restrictions of the "optimistic" mode, you can set it to the "optimistic" mode.
meta-schema: "dm_meta"          # The downstream database that stores the `meta` information.
timezone: "Asia/Shanghai"       # The timezone.
case-sensitive: false           # Determines whether the schema/table is case-sensitive.
online-ddl: true                # Supports automatic processing of upstream "gh-ost" and "pt".

> online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` will be deprecated in the future, so it is recommended to use `online-ddl`.

Reviewer: has been deprecated in v2.0.6

> online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` will be deprecated in the future, so it is recommended to use `online-ddl`.

Author: ```suggestion
online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` has been deprecated in v2.0.6, so it is recommended to use `online-ddl`.
```

# Configuration arguments of the sync processing unit.
syncers:
  global:                            # The configuration name of the processing unit.

>     worker-count: 16                 # The number of concurrent threads that read, parse, and apply binlog events in the sync processing unit (16 by default). Adjusting this value has no effect on the upstream, but has a significant effect on the downstream database.

Reviewer 2: ```suggestion
    worker-count: 16                 # The number of concurrent threads that read, parse, and apply binlog that has been transferred to the local(16 by default). Adjusting this value has no effect on the upstream, but has a significant effect on the downstream database.
```

>     worker-count: 16                 # The number of concurrent threads that read, parse, and apply binlog events in the sync processing unit (16 by default). Adjusting this value has no effect on the upstream, but has a significant effect on the downstream database.

Reviewer: Not read, parse, and apply. Only apply

>     worker-count: 16                 # The number of concurrent threads that apply binlogs which has been transferred to the local (16 by default). Adjusting this parameter has no effect on the concurrency of upstream pull logs, but has a significant effect on the downstream database.

Reviewer 3: ```suggestion
    worker-count: 16                 # The number of concurrent threads that apply binlogs which have been transferred to the local (16 by default). Adjusting this parameter has no effect on the concurrency of upstream pull logs, but has a significant effect on the downstream database.
```
